### PR TITLE
*: support varchar only in ddl

### PR DESCRIPTION
**Pull request by cyliu0**

### CODE REVIEW

File: pkg/util/hack.go [hunk 1]
```diff
@@ -28,3 +30,11 @@ func Slice(s string) (b []byte) {
 	pbytes.Cap = pstring.Len
 	return
 }
+
+// VarcharOnly converts char to varchar if ENV VAR VARCHAR_ONLY=true
+func VarcharOnly(query string) string {
+	if os.Getenv("VARCHAR_ONLY") == "true" {
+		return strings.Replace(strings.Replace(query, " CHAR", " VARCHAR", -1), " char", " varchar", -1)
```
Comment (fuyufjh) [inline pkg/util/hack.go]: Emmm this looks stupid. Why not just replacing them in code?

Comment (cyliu0) [reply pkg/util/hack.go]: In case we need to support the char type in the future we don't need to modify any code.

Comment (fuyufjh) [reply pkg/util/hack.go]: In that case I would rather modify the code 😄

Comment (fuyufjh) [reply pkg/util/hack.go]: I prefer not to introduce any environment variable here. Too tricky and looks like a pitfall in future.

**Review by lmatz — APPROVED** [review]
LGTM

**Pull request merged**